usaspending_api/settings.py

```diff
@@ -48,25 +45,34 @@
 
 # AWS locations for CSV files
 CSV_LOCAL_PATH = os.path.join(BASE_DIR, 'csv_downloads', '')
-
-DOWNLOAD_ENV = ''
+DOWNLOAD_ENV = ""
 BULK_DOWNLOAD_LOCAL_PATH = os.path.join(BASE_DIR, 'bulk_downloads', '')
+
 BULK_DOWNLOAD_S3_BUCKET_NAME = ""
 BUCK_DOWNLOAD_S3_REDIRECT_DIR = "generated_downloads"
 BULK_DOWNLOAD_SQS_QUEUE_NAME = ""
 MONTHLY_DOWNLOAD_S3_BUCKET_NAME = ""
 MONTHLY_DOWNLOAD_S3_REDIRECT_DIR = "award_data_archive"
 BROKER_AGENCY_BUCKET_NAME = ""
 FPDS_BUCKET_NAME = ""
+if not FPDS_BUCKET_NAME:
+    FPDS_BUCKET_NAME = os.environ.get('FPDS_BUCKET_NAME')
+DELETED_TRANSACTIONS_S3_BUCKET_NAME = ""
+if not DELETED_TRANSACTIONS_S3_BUCKET_NAME:
+    DELETED_TRANSACTIONS_S3_BUCKET_NAME = os.environ.get('DELETED_TRANSACTIONS_S3_BUCKET_NAME')
+STATE_DATA_BUCKET = ""
+if not STATE_DATA_BUCKET:
+    STATE_DATA_BUCKET = os.environ.get('STATE_DATA_BUCKET')
 CFDA_BUCKET_NAME = ""
 CFDA_REGION = ""
 CFDA_FILE_PATH = ""
 ROSETTA_DICT_S3_PATH = "da-public-files/user_reference_docs/DATA Transparency Crosswalk.xlsx"
 
 # Elasticsearch
 ES_HOSTNAME = ""
+if not ES_HOSTNAME:
+    ES_HOSTNAME = os.environ.get('ES_HOSTNAME')
 TRANSACTIONS_INDEX_ROOT = os.environ.get('ES_TRX_ROOT') or 'future-transactions'
-DOWNLOAD_QUERY_SIZE = 500000
```

Comment:
No need to duplicate `MAX_DOWNLOAD_LIMIT`, removing `DOWNLOAD_QUERY_SIZE` which was only used by the elasticsearch helper file.